html.editor/src/org/netbeans/modules/html/editor/resources/DTDs/3_2/lat1.ent

```diff
@@ -1,3 +1,51 @@
+<!--
+
+W3C Software and Document Notice and License
+
+This work is being provided by the copyright holders under the following license.
+
+License
+
+By obtaining and/or copying this work, you (the licensee) agree that you have
+read, understood, and will comply with the following terms and conditions.
+
+Permission to copy, modify, and distribute this work, with or without
+modification, for any purpose and without fee or royalty is hereby granted,
+provided that you include the following on ALL copies of the work or portions
+thereof, including modifications:
+
+- The full text of this NOTICE in a location viewable to users of the
```

Comment:
So we also need something for NOTICE?

Reply:
We already ship the W3C license and it is placed in a location viewable by the user (the license directory). This should cover it from my understanding.